packages/cli/src/commands/generate/templates/cell/test.js.template

```diff
@@ -1,10 +1,31 @@
-import React from 'react';
-import { fireEvent, cleanup } from '@testing-library/react'
+import { render, cleanup } from '@testing-library/react'
 
-import ${pascalName}Cell from './${pascalName}Cell'
+import { Loading, Empty, Failure, Success } from './${camelName}Cell'
 
-describe('${pascalName}Cell', () => {
-  it('is true', () => {
-    expect(true).toBe(true)
+const error = { message: 'error message' }
+
+describe('${camelName}Cell', () => {
+  afterEach(() => {
+    cleanup()
+  })
+  it('Loading renders successfully', () => {
+    expect(() => {
+      render(<Loading />)
+    }).not.toThrow()
+  })
+  it('Empty renders successfully', () => {
+    expect(() => {
+      render(<Empty />)
+    }).not.toThrow()
+  })
+  it('Failure renders successfully', () => {
+    expect(() => {
+      render(<Failure error={error} />)
```

Comment:
Will `error` actually be defined here? You should be able to just stub this out with something like:

    <Failure error={{message: 'problem'}} />

A generated cell will try to render `error.message`

Reply:
ah, that’s much better. thanks.